README.md

```diff
@@ -0,0 +1,331 @@
+                wpan-tools tcpdump uhttpd uboot-envtools \
+                alsa-lib alsa-utils alsa-utils-tests
+
+    endef
+
+
+Please ensure to remove .config, tmp folders before running make menuconfig.
+
+    $ make menuconfig
+
+Verify that package name is selected. Just save and exit.
+
+## Customising your linux kernel
+
+Kernel configuration is saved at target/linux/pistachio/config-4.1
+e.g. This the one for the IMG pistachio board as it has some useful defaults.
+
+To customise to suit your requirements use the following command:
+
+    $ make kernel_menuconfig
+
+If you change any option you need and then save & quit GUI, the changed configuration will get written into target/linux/pistachio/config-4.1
+
+For more details please refer [OpenWrt Build System]("http://wiki.openwrt.org/doc/howto/build")
+
+## Serial console
+Connect the Marduk board to development computer over serial port. Open a serial console on host PC.
+
+- In Windows PuTTY, HyperTerminal, or any other free application can be used.
+
+- In Linux, the simplest option is to use the miniterm.py application that comes bundled with Python.
+
+First work out which device you want, you can see them by running:
+
+    $ ls /dev/ttyUSB*
+
+Then connect to it (remember to choose the correct device listed in above command):
+
+    $ sudo miniterm.py /dev/ttyUSB0 -b 115200
+
+## Instructions for putting root filesystem on usb and booting from usb drive
+Firstly you need to format USB drive into ext4 parition. Partition your media with one big ext4 partion (use dos partition table)
+You can check the partition number by following way:
+
+
+`$ mount | grep /media
+/dev/sdx1 on /media/user/43e3311e-7b95-4693-bfcd-b07fa4590a0d type ext4 (rw,nosuid,nodev,uhelper=udisks2)`
+
+Alternatively, you can also check the output of the dmesg command.
+
+Next you need to ensure that it has a DOS partition table containing a Linux ext4 partition.
+<b>IMPORTANT! Make sure that you replace /dev/sdx for the right device name, e.g. /dev/sdb</b>
+
+Unmount the filesystem before creating new partition table:
+
+    $ sudo umount /dev/sdx?
+
+Create new partition table:
+
+    $ sudo fdisk /dev/sdx
+    > enter 'o' to create a new empty DOS partition table
+    > enter 'n' to create a new partition; just press return at every prompt to accept the default values
+    > enter 'w' to write the new partition table and exit
+    $ sudo mkfs.ext4 /dev/sdx1
+
+Mount the usb drive:
+
+    $ sudo mount /dev/sdx1 /mnt/
+
+To put filesystem on USB you will need 'openwrt-pistachio-marduk-marduk_cc2520-rootfs.tar.gz' which is available at "openwrt/bin/pistachio"
+
+Extract the openwrt-pistachio-marduk-marduk_cc2520-rootfs.tar.gz onto the partition you just created
+
+    # sudo rm -rf /mnt/*
+    # sudo tar -xf bin/pistachio/openwrt-pistachio-marduk-marduk_cc2520-rootfs.tar.gz -C /mnt/
+    # sudo umount /mnt/
+
+
+Run "sync" command to synchronize the data properly on the USB drive.
+
+    # sync
```

Comment:
Replace # with $ for consistency with other commands in this file to be run on developer's PC